lib/src/components/NxFileUpload/NxFileUpload.scss

```diff
@@ -8,7 +8,7 @@
 @use '../../scss-shared/nx-text-helpers';
 @use '../../scss-shared/nx-container-helpers';
 
-.nx-file-upload {
+.nx-file-upload, .nx-multi-file-upload {
```

Comment:
> It is interesting that these styles mostly end up reused. Let's just add the `.nx-file-upload` classes to the new component rather than duplicating them all with `.nx-multi-file-upload` classes. Of course we will also need `.nx-multi-file-upload` itself and the classes that are exclusive to the new component, for the block of new styles below (which should probably be in their own file)

I applied the `nx-file-upload` class on the component and added a new stylesheet here: [1cfce3b](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/1cfce3b426e32c563ab6abae4e0136ba7888cc49). The additional CSS fixes in other comments to come in the next commits to make it easier to parse through

Reply:
> or changing their classNames in the html back to .nx-file-upload

That's what I meant to happen

Reply:
> > or changing their classNames in the html back to .nx-file-upload
> 
> That's what I meant to happen

Fixed here: [ac76a50](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/ac76a50d72f990b203db8694c5c079c2c340dd3c)